lightning/src/ln/channel.rs

```diff
@@ -1656,6 +1658,58 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 		cmp::min(self.value_to_self_msat as i64 - self.get_outbound_pending_htlc_stats().1 as i64, 0) as u64)
 	}
 
+	fn commit_tx_fee_per_htlc_msat(&self) -> u64 {
```

Comment:
Hmm, this is easy to misuse, no? Technically using this directly isn't gonna round correctly (N * round(X) != round(N * X)). It may pass for tests, but in general I kinda prefer exposing the constants and doing the math directly in the test.

Reply:
Gotcha. By that logic, should the fee_spike_reserve "HTLCs" be included in the same calculation, as well? or can it still be separate?